### PR TITLE
Fixes keypoints in lesson content and infrastructure episodes

### LINKED ISSUE #431: Keypoint needs to be moved
### How could the content be improved?

> - The main pages of a lesson website are created from Markdown or RMarkdown files in the `episodes` folder.

This keypoint needs to be moved to the adding lesson content episode as it is currently in the infrastructure episode

### Which part of the content does your suggestion apply to?

https://carpentries.github.io/lesson-development-training/instructor/lesson-content.html

### PR DESCRIPTION
**Pull request by anenadic**
Fixes #431.

### CODE REVIEW

File: episodes/lesson-content.md [hunk 1]
```diff
@@ -72,7 +72,7 @@ from how it appears on your website, or not at all.
 
 We will encounter more examples of this throughout the lesson,
 but the important thing to remember for now is that
-**how GitHub displays source (R) Markdown files is not a reliable indicator
+**how GitHub displays source (R)Markdown files is not a reliable indicator
```
Comment (tobyhodges) [inline episodes/lesson-content.md:75]:
```suggestion
**how GitHub displays source (R) Markdown files is not a reliable indicator
```

I think the space between the R and the M is [how we are supposed to write it](https://rmarkdown.rstudio.com/).

File: episodes/lesson-content.md [hunk 2]
```diff
@@ -102,9 +102,9 @@ as a bullet list in the `index.md` file of your lesson repository.
 
 ## Episodes
 
-As we've discussed previously, the main body of the lesson is written in _episodes_:
+The main body of the lesson is written in _episodes_:
 the individual chunks or sections that the lesson is separated into.
-The episode pages of the lesson site will be constructed from Markdown files
+The episode pages of the lesson site will be constructed from Markdown or RMarkdown files
```
Comment (tobyhodges) [inline episodes/lesson-content.md:107]:
```suggestion
The episode pages of the lesson site will be constructed from Markdown or R Markdown files
```

File: episodes/lesson-content.md [hunk 3]
```diff
@@ -491,7 +491,7 @@ find and fix the problem when you notice the build process fail.
 
 :::::::::::::::::::::::::::::::::::::::: keypoints
 
-- Lesson episodes are individual Markdown pages in a lesson repository.
+- The main pages of a lesson website (lesson episodes) are created from individual Markdown or RMarkdown files in the `episodes` folder of a lesson repository
```
Comment (tobyhodges) [inline episodes/lesson-content.md:494]:
```suggestion
- The main pages of a lesson website (lesson episodes) are created from individual Markdown or R Markdown files in the `episodes` folder of a lesson repository
```

**Review by tobyhodges — CHANGES_REQUESTED** [review]
Looks good to me, except that I think we should use "R Markdown"

**Review by sstevens2 — APPROVED** [review]
Looks good to me! once Toby's comments are addressed

File: episodes/lesson-content.md (hunk 1 above)
Comment (anenadic) [reply episodes/lesson-content.md:75]: Ah, I think we may then need to change this elsewhere for consistency. Thanks @tobyhodges.

**Comment by anenadic** [comment]
All addressed now I think @tobyhodges @sstevens2.

**Review by tobyhodges — APPROVED** [review]

**Pull request merged**